clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -210,6 +210,18 @@ public void testInvalidSocketReceiveBufferSize() {
         new KafkaConsumer<>(config, new ByteArrayDeserializer(), new ByteArrayDeserializer());
     }
 
+    @Test
+    public void testInitSuccessWithDefaultNullGroupIdAndValidGroupInstanceId() {
+        Map<String, Object> config = new HashMap<>();
+        config.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9999");
+        config.put(ConsumerConfig.SEND_BUFFER_CONFIG, Selectable.USE_DEFAULT_BUFFER_SIZE);
+        config.put(ConsumerConfig.RECEIVE_BUFFER_CONFIG, Selectable.USE_DEFAULT_BUFFER_SIZE);
```

Comment:
Why do we need to set `SEND_BUFFER_CONFIG` and `RECEIVE_BUFFER_CONFIG` ?

Reply:
No we don't lol